plugin.video.fantasticc/default.py

```diff
@@ -344,19 +344,28 @@ def INDEXCOLLECT(url):   # Index Collections Pages
     xbmc.log('URL Loading: %s' % url)
     html = get_html(url)
 
-    match = re.compile(
-        '<a class="clnk" href="([^"]+)">([^<]+)</a>.+?"counter-right">'
-        '(\d+) videos.+?style="background: url\(([^)]+)\);',
-        re.DOTALL).findall(html)
-    for gurl, name, num_of_vids, icons in match:
+    match = re.compile('<a class="clnk" href="(.+?)">(.+?)</a>(.+?)<div class="tag-list">', re.DOTALL).findall(html)
+
+    for gurl, name, chtml in match:
         xbmc.log('Name [%s]' % name)
         realurl = 'http://fantasti.cc%s' % gurl
         name = unescape(name)
         mode = 1
-        if num_of_vids == str(0):
-            continue
 
-        addDir(name + ' (' + num_of_vids + ' vids)', realurl, mode, icons)
+        #scrape number of vids
+        vidnumber = re.search('"videosListNumber"><b>(.*?)<',chtml)
```

Comment:
I have updated the code by breaking the long line into two
Noticed there are a few more lines which are longer than the one that I added. Have left them alone as I didnt add them